routers/user/home.go

```diff
@@ -314,23 +321,27 @@ func showOrgProfile(ctx *middleware.Context) {
 	org := ctx.Org.Organization
 	ctx.Data["Title"] = org.FullName
 
-	repos, err := models.GetRepositories(org.Id, ctx.IsSigned && (ctx.User.IsAdmin || org.IsOrgMember(ctx.User.Id)))
-	if err != nil {
-		ctx.Handle(500, "GetRepositories", err)
-		return
+	if ctx.IsSigned {
+		if err := org.GetUserRepositories(ctx.User.Id); err != nil {
+			ctx.Handle(500, "GetUserRepositories", err)
+			return
+		}
+		ctx.Data["Repos"] = org.Repos
+	} else {
+		if repos, err := models.GetRepositories(org.Id, false); err != nil {
+			ctx.Handle(500, "GetRepositories", err)
+			return
+		} else {
```

Comment:
You could also do:

```go
repo, err := org.GetUserRepositories(ctx.User.Id);
if err != nil {
    ctx.Handle(500, "GetUserRepositories", err)
    return
}
ctx.Data["Repos"] = repos
```

This block lives inside another else block, so `that do exist outside of the area where they are used.` isn't true.